### PR TITLE
small changes to README and deploy command

### PR DESCRIPTION
**Pull request by tbeseda**
Update `arc env` commands to new syntax.
Fix a couple typos.

### CODE REVIEW

**Review by mcansh — APPROVED** [review]
looks great! thanks Taylor!

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Prior to your first deployment, you'll need to do a few things:
 
 You can find the CloudFormation template that Architect generated for you in the sam.yaml file.
 
-To find it on AWS, you can search for [CloudFormation](https://console.aws.amazon.com/cloudformation/home) (make sure you're looking at the correct region!) and find the name of your stack (the name is a PascalCased version of what you have in `app.arc`, so by default it's RemixAwsStackStaging) that matches what's in `app.arc`, you can find all of your app's resources under the "Resources" tab.
+To find it on AWS, you can search for [CloudFormation](https://console.aws.amazon.com/cloudformation/home) (make sure you're looking at the correct region!) and find the name of your stack (the name is a PascalCased version of what you have in `app.arc`, so by default it's similar to RemixGrungeStackStaging) that matches what's in `app.arc`, you can find all of your app's resources under the "Resources" tab.
```
Comment (mcansh) [inline README.md:79]: @kentcdodds we could do something similar in the init script to generate the CloudFormation name 😉

Comment (mcansh) [reply README.md:79]: update: added here :) #17

Comment (tbeseda) [reply README.md:79]: nice good call

Comment (mcansh) [reply README.md:79]: the more i use `remix.init` the more i love it 😄

**Review by kentcdodds — APPROVED** [review]
Stellar, thank you 👏👏

**Pull request merged**